Detections/SecurityEvent/PotentialRemoteDesktopTunneling.yaml

```diff
@@ -0,0 +1,37 @@
+id: d2e8fd50-8d66-11ec-b909-0242ac120002
+name: Potential Remote Desktop Tunneling
+description: |
+   'This query detects remote desktop authentication attempts with a localhost source address which can indicate a tunneled login.
+   Ref: https://www.mandiant.com/resources/bypassing-network-restrictions-through-rdp-tunneling'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandandControl
+relevantTechniques:
+  - T1572
+query: |
+ SecurityEvent
+    | where EventID in (4624,4625) and LogonTypeName in (10) and IpAddress in ("::1","127.0.0.1")
```

Comment:
Wondering if you wanted LogonType vs LogonTypeName here
![image](https://user-images.githubusercontent.com/37783395/154015383-889977d2-8138-4c1d-8818-602a9340be0b.png)